src/components/measure/MeasureDirective.js

```diff
@@ -43,7 +43,7 @@ goog.require('ga_measure_service');
                 update(evt.target);
               });
             });
-            newFeature.changed();
+            update(newFeature);
```

Comment:
strange that this stopped working with an ol3 update. @oterral @fredj Do you know the reason?